### DISCUSSION TITLE
크롬웹스토어에 있는 백신 매크로 초기화 안 되는 문제 문의

### DISCUSSION THREAD

**Discussion by sungdansy** [changdoc/naver-vaccine-macro#157] [Q&A]
안녕하세요.
회사에서 크롬 웹스토어에 있는 백신 매크로를 설치해서 이용하고 있습니다.

초기화 하고 다시 데이터를 넣을려고 하는데 초기화 버튼이 동작을 하지 않습니다.

해결 하는 방법이 있나요?

**Comment by changdoc** [comment]
초기화는 텔레그램 관련 정보간 제거하게 되어 있고요.
graphql쪽 데이터를 다시 넣고 싶으신거면 내용을 교체 하시고 병원목록 생성 하시면 됩니다.
